app/src/main/resources/com/oracle/javafx/scenebuilder/app/menubar/MenuBar.fxml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <!--
-  Copyright (c) 2016, 2019, Gluon and/or its affiliates.
+  Copyright (c) 2016, 2019,2021, Gluon and/or its affiliates.
```

Comment:
This needs to be "2016, 2021, Gluon..."

Reply:
Is this basically not the year of changes but year of inception and last change? Didnt knew that.

Reply:
Yes :smile: